.eslintrc

```diff
@@ -1,24 +1,24 @@
-{
-  "extends": [
-    "eslint:recommended",
-    "plugin:@typescript-eslint/recommended",
-    "plugin:jest/recommended",
-    "plugin:prettier/recommended"
-  ],
-  "env": {
-    "node": true,
-    "es6": true
-  },
-  "parserOptions": {
-    "project": "./tsconfig.json"
-  },
-  "rules": {
-    "@typescript-eslint/no-explicit-any": "off",
-    "@typescript-eslint/no-use-before-define": [
-      "error",
-      {
-        "functions": false
-      }
-    ]
-  }
-}
+// {
```

Comment:
Making a follow up PR to update linting due to the introduction of @jupiterone/typescript-tools.  Didn't want to muddy up this functionality